### ISSUE TITLE
Terraform apply taking ages

### ISSUE THREAD

**Issue by gmanjunath** [terraform-aws-modules/terraform-aws-eks#1331]
## Description

First of all I would like to  say **big thanks** to contributors who have taken time to implement such a wonderful module. Much appreciated. I'm trying to use the example as it is and I'm noticing terraform apply is not deploying any resource, basically nothing happens for **almost 20+ mins and then I stop by pressing ctrl-c**. I did retry couple of times and still the same issue, infact I tried using different version of terraform (v0.12.31, v0.13.1, v0.14.0), still no luck. I literally deleted and re-initialised.

I have used other modules such as vpc and could successfully create VPC and EC2 instances. I'm trying to spin up EKS cluster in **EU-west-2 region.**

## ⚠️ Note

Before you submit an issue, please perform the following first:

1. Remove the local `.terraform` directory (! ONLY if state is stored remotely, which hopefully you are following that best practice!): `rm -rf .terraform/`
2. Re-initialize the project root to pull down modules: `terraform init`
3. Re-attempt your terraform plan or apply and check if the issue still persists

## Versions

 - Terraform:
0.12.31, 0.13.1, 0.14.0
 - Provider(s):
AWS
 - Module:
EKS & VPC

## Reproduction

Are you using workspaces? No
Have you cleared the local cache (see Notice section above)?  Yes
List steps in order that led up to the issue you encountered terraform init followed by terraform apply

### Code Snippet to Reproduce
 I'm using basic example from the [git hub](https://github.com/terraform-aws-modules/terraform-aws-eks/tree/master/examples/basic)

## Expected behavior
 
I'm expecting terraform apply to spin up the resources in my AWS account and create EKS cluster.

## Actual behavior

Successfully initialised terraform, however terraform apply script just doesn;t do anything, I can control -C and it quits. Its not like hanging, because I could ctrl-c. I waited for almost 20+ mins, but no response/logs appear in the console.

### Terminal Output Screenshot(s)

base) Manjunaths-MBP:learn-terraform-modules manjunathg$ terraform init
Initializing modules...

Initializing the backend...

Initializing provider plugins...
- Reusing previous version of hashicorp/template from the dependency lock file
- Reusing previous version of hashicorp/local from the dependency lock file
- Reusing previous version of hashicorp/null from the dependency lock file
- Reusing previous version of hashicorp/random from the dependency lock file
- Reusing previous version of hashicorp/aws from the dependency lock file
- Reusing previous version of hashicorp/kubernetes from the dependency lock file
- Installing hashicorp/local v2.1.0...
- Installed hashicorp/local v2.1.0 (self-signed, key ID 34365D9472D7468F)
- Installing hashicorp/null v3.1.0...
- Installed hashicorp/null v3.1.0 (self-signed, key ID 34365D9472D7468F)
- Installing hashicorp/random v3.1.0...
- Installed hashicorp/random v3.1.0 (self-signed, key ID 34365D9472D7468F)
- Installing hashicorp/aws v3.38.0...
- Installed hashicorp/aws v3.38.0 (self-signed, key ID 34365D9472D7468F)
- Installing hashicorp/kubernetes v1.13.3...
- Installed hashicorp/kubernetes v1.13.3 (self-signed, key ID 34365D9472D7468F)
- Installing hashicorp/template v2.2.0...
- Installed hashicorp/template v2.2.0 (self-signed, key ID 34365D9472D7468F)

Partner and community providers are signed by their developers.
If you'd like to know more about provider signing, you can read about it here:
https://www.terraform.io/docs/plugins/signing.html

Terraform has been successfully initialized!

You may now begin working with Terraform. Try running "terraform plan" to see
any changes that are required for your infrastructure. All Terraform commands
should now work.

If you ever set or change modules or backend configuration for Terraform,
rerun this command to reinitialize your working directory. If you forget, other
commands will detect it and remind you to do so if necessary.
(base) Manjunaths-MBP:learn-terraform-modules manjunathg$ terraform apply

**Comment by gmanjunath** [comment]
I think I figured it out, why terraform apply was taking time. Basically I started commenting the code and working towards the minimal set of code and finally at one point terraform complained about credentials. I did pass the credentials as part of variables.tfvar, but still it was complaining. After some googling I found this link https://github.com/hashicorp/terraform-provider-aws/issues/13057, I followed the steps by exporting the variables. Once the variables were exported, I could see terraform spinning up and creating the resources.
